GDJS/Runtime/events-tools/runtimescenetools.ts

```diff
@@ -326,6 +326,36 @@ namespace gdjs {
       ): boolean => {
         return runtimeScene.getGame().hasScene(sceneName);
       };
+
+      /**
+       * Preload a scene assets as soon as possible in background.
+       */
+      export const prioritizeLoadingOfScene = (
+        runtimeScene: gdjs.RuntimeScene,
+        sceneName: string
+      ): void => {
+        runtimeScene.getGame().prioritizeLoadingOfScene(sceneName);
+      };
+
+      /**
+       * @return The progress of assets loading in background for a scene (between 0 and 1).
+       */
+      export const getSceneLoadingProgress = (
```

Comment:
For GDevelop users, would it be simpler to return a percentage?

```suggestion
      export const getSceneLoadingProgressPercent = (
```

Reply:
A percentage is a format not a value `0.1` can be formatted `"10%"`. Values for 0 to 1 is easier to use because you don't have to divided by 100 every time you use it, for instance to calculate the position of the right side of the bar.

Reply:
Ok, fine to keep it then, even if we have percentages in other part of the app (sounds volume, etc...).